lib/ruby_lsp/listeners/definition.rb

```diff
@@ -145,6 +147,19 @@ def handle_require_definition(node)
         end
       end
 
+      sig { params(node: Prism::CallNode).void }
+      def handle_autoload_definition(node)
+        arguments = node.arguments&.arguments
+        return unless arguments
```

Comment:
To fix the type checking error.
```suggestion
        arguments = node.arguments&.arguments&.first
        return unless arguments.is_a?(Prism::SymbolNode)
```